german/ruderboot.md

```diff
@@ -1,4 +1,4 @@
-Sie wachen von der Sonne geblendet auf und stellen ziemlich benommen fest, dass Sie allein 
```

Comment:
Was deleting the word "allein" intentional? (I don't speak German, so I can't tell whether the change makes sense.) The other two "modifications" look like they didn't actually change anything, so if this one was intentional I'll be happy to merge this.